### ISSUE TITLE
mypy checks weren't run until late February. Started failing in CI

### ISSUE THREAD

**Issue by BertR** [jupyter/nbgrader#1328]
`pytest-mypy` was not compatible with `pytest-xdist` until the 0.5.0 release on 23rd of February 2020 (https://pypi.org/project/pytest-mypy/#history and https://github.com/dbader/pytest-mypy/blob/master/changelog.md)

This means the mypy work never actually ran / passed.
My suggestion would be to pint pytest-mypy to a previous version
for now, and then fix the type annotation in a new PR.

This should unblock some of the PRs currently failing.
